vendor/github.com/boltdb/bolt/bolt_windows.go

```diff
@@ -89,7 +89,7 @@ func flock(db *DB, mode os.FileMode, exclusive bool, timeout time.Duration) erro
 func funlock(db *DB) error {
 	err := unlockFileEx(syscall.Handle(db.lockfile.Fd()), 0, 1, 0, &syscall.Overlapped{})
 	db.lockfile.Close()
-	os.Remove(db.path+lockExt)
+	os.Remove(db.path + lockExt)
 	return err
 }
 
```

Comment:
Looks like you may have run `go fmt` on the vendor directory. While annoying we need to leave the vendor code unmodified from upstream. Can you revert these vendor changes?